openapi/src/parts/controlled_aws_s3_storage_folder.yaml

```diff
@@ -123,7 +123,7 @@ paths:
       tags: [ ControlledAwsResource ]
       responses:
         '200':
```

Comment:
Do you expect that this will break the CLI without updating the WSM client?  I agree with the change, just want to get ahead of it in the startup script if necessary.

Reply:
verified on cli / axon - not a breaking change